GameOfSwift3/GameOfSwift3/Player.swift

```diff
@@ -15,21 +15,61 @@ class Player {
     private var name = ""
     
     func nameYourself() {
-        print("Hello player, please enter your name: ")
+        print("\n Hello player, please enter your name: ")
         if let name = readLine(),
             name != "" {
-            print("Welcome \(name) !")
+            print(" Welcome \(name) !\n")
             self.name = name
         } else  {
             print("error in naming, try again:")
             nameYourself()
         }
     }
     
-    func resume() {
-        print("\(name)'s team:")
+     func resume() {
+        print("\n \(name)'s team:")
         for character in team {
-            print("\(character.type) \(character.name) who start with \(character.maxLifePoint) lifepoints and \(character.tools) in his hand")
+            print("\n\(character.type) \(character.name) who start with \(character.maxLifePoint) 💜 and \(character.tools.name) in his hand")
         }
     }
+    
+    func showTeam() {
+        for (index, character) in team.enumerated() {
+            if character.lifePoint > 0 {
+                print("\n\(index): \(character.name) the \(character.type): \(character.lifePoint) 💜, \(character.tools.name)")}
+        }
+    }
+    
+    func chooseCharacter() -> Character {
+        print("\nChoice:")
+            if let choice = readLine(),
+            choice != "",
+            let indexChoice = Int(choice),
+            indexChoice >= 0,
+            indexChoice < team.count {
+            let characterChoice = team[indexChoice]
+                return team[indexChoice]
+            } else {
+                print("You have to choose number")
+                return chooseCharacter()
+            }
+        }
+    
+        func isTeamAlive() -> Bool {
+        let isTeamAlive = team.contains { character in character.isAlive()}
+        if isTeamAlive == true {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    func winner() {
```

Comment:
A method do something, it's an action so the name should be a verb. You should find another name that actually represent what the method does.